xarray/core/dataarray.py

```diff
@@ -1194,13 +1194,22 @@ def to_dict(self):
         Convert this xarray.DataArray into a dictionary following xarray
         naming conventions.
 
-        Useful for coverting to json.
+        Useful for coverting to json.To avoid datetime incompatibility
```

Comment:
Missing a space after `json`.

Maybe add "Converts all variables and attributes to native Python objects."